apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -171,98 +212,151 @@ static int create_file(char *path, ssize_t bufsize, file_type_t type)
 	return ret;
 }
 
-static int verify_random_file(char *path, int bufsize, int *index)
+static int create_sample_text_file(void)
+{
+	int ret;
+	int fd_sample;
```

Comment:
If it created already, we can skip it right? or...you want create it every time?
For both of 2 cases, we need extra process.
If we keep created file, then we should check that file is exist and size of file with 'stat' & 'stat.st_size' .
or if we create newly every time, at least O_TRUNC is needed.. or unlink.

Reply:
Updated